src/servers/grpc_server.cc

```diff
@@ -311,24 +391,28 @@ class InferBaseContext : public BaseContext<LifeCycle, AsyncResources> {
       if (err == nullptr) {
         err = GRPCToInput(request.meta_data(), request, request_provider);
```

Comment:
GRPCToInput should handle the shared memory case in addition to the existing raw case.

Reply:
Done